tests/Feature/PhotosTest.php

```diff
@@ -202,6 +198,56 @@ public function testUpload()
 		AccessControl::logout();
 	}
 
+	/**
+	 * Test live photo upload.
+	 *
+	 * @return void
+	 */
+	public function testLivePhotoUpload()
+	{
+		$photos_tests = new PhotosUnitTest($this);
+
+		AccessControl::log_as_id(0);
+		// MUST use exiftool to get live photo metadata
+		$init_config_value = Configs::get_value('has_exiftool');
+		Configs::set('has_exiftool', '1');
```

Comment:
What happens if exiftool is not actually available? I guess the test will crash, doesn't it?

Reply:
yes. But in our test suite we do install exiftool.

Reply:
Yes, but the CI at the server side is not the only place where tests are run. Maybe be there are developers which run the test suite locally on their machine, but haven't installed exiftool. Then the tests crash. Therefore I would not change the configuration setting, hut skip the test if exiftool is not available and spit out a warning that not all tests could be run due to missing dependencies.